server/market/bookrouter.go

```diff
@@ -46,7 +47,7 @@ type bookUpdateSignal struct {
 // BookSource is a source of a market's order book and a feed of updates to the
 // order book and epoch queue.
 type BookSource interface {
-	Book() (buys []*order.LimitOrder, sells []*order.LimitOrder)
+	Book() (epoch int64, buys []*order.LimitOrder, sells []*order.LimitOrder)
```

Comment:
This may prove to be unnecessary since it will be zero before the market starts and a `newEpochAction` signal will come just before new epoch orders begin to be accepted.